features/organization/children.feature

```diff
@@ -70,7 +85,17 @@ Feature:
         And I should see "La structure a été mise à jour avec succès."
         And I should see "Lorem ipsum"
 
-    Scenario: As an organization, I cannot update an organization I don't have access to
-        Given I am authenticated as "DT75"
-        When I go to "/organizations/201/children/202/edit"
+    Scenario: As an admin of an organization, I cannot update an organization I don't have access to
+        Given I am authenticated as "john.doe@resop.com"
+        When I go to "/organizations/201/children/204/edit"
+        Then the response status code should be 403
+
+    Scenario: As anonymous, I cannot update an organization
```

Comment:
Are you sure we need to run all those security tests every time ? We could only test anonymous & admins without password access on once, in a dedicated feature file